account_partner_company_group/readme/CREDITS.md

```diff
@@ -0,0 +1 @@
+The migration of this module from 17.0 to 18.0 was financially supported by Camptocamp.
```

Comment:
I don't see you add your name to contributors.txt. Your contribution is worth for future generations to admire. :smile:

Reply:
My contribution is not much enough for adding my name to the contributors. so, let it as it be.